### PR TITLE
Feature/mobile adaptation

### PR DESCRIPTION
**Pull request by Louis-Lesage**
Touch screen feature
Leave comments if you feel like
Thanks

### CODE REVIEW

File: src/ControlMobile.css [hunk 1]
```diff
@@ -0,0 +1,240 @@
+
+ul, li {
```
Comment (bfirsh) [inline src/ControlMobile.css]: This should be scoped to the controller. Or, just use divs?

File: src/ControlMobile.css [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  list-style: none;
+}
+/* controller global */
+.controller {
+  position: relative;
+  -webkit-3d-transform: rotate(0deg) scale(1);
+}
+/* GBASP BASE */
```
Comment (bfirsh) [inline src/ControlMobile.css]: Why is there a separate `.controller` and `.gbasp` class? What does `.gbasp` mean?

**Comment by Louis-Lesage** [comment]
Hi, thank you for reviewing.
I will implement nipple.js in the projet and i will look to get it more responsive.

**Comment by memalign** [comment]
Thanks for the hard work! I’d love to see this support land!

**Comment by Louis-Lesage** [comment]
I'm quite busy at the moment, so if anyone wanna jump on the feature feel free!

**Comment by antris** [comment]
I tried looking into this but the branch has diverged so much from master that I found it easier to start on my own code. I got some basic controls working, but still have lot of work to do. I can push my WIP branch if anyone wants to take a look.

**Pull request closed without merging**